src/transform/src/join_implementation.rs

```diff
@@ -268,10 +270,24 @@ impl JoinImplementation {
                 }
                 let push_down_characteristics =
                     FilterCharacteristics::filter_characteristics(&push_downs[index])?;
-                // let push_down_factor = push_down_characteristics.worst_case_scaling_factor();
```

Comment:
Is there a reason why `push_down_characteristics` is not guarded by the below `if features.enable_cardinality_estimates`?

Reply:
I misunderstood this on our earlier call. These lines go back earlier, though the worst-case scaling factor was added (unconditionally) on the original cardinality PR. #25784 removed the worst-case scaling factor, but not for any particular reason I can see.

Reply:
Sorry, I meant the new version of the line, which is not commented out. Could we move it inside the below `if`?

Reply:
Sure! I don't think it depends on cardinality at all, though.

Reply:
Thanks!